gulpfile.js

```diff
@@ -432,6 +432,14 @@ gulp.task('clean', function (done) {
 
 gulp.task('build', gulp.series('clean', 'ext:build', 'ext:install-service'));
 
+gulp.task('quick-build', function(done) {
```

Comment:
How do we know if out contains the latest code in the extension? It can also be stale code out, right? Why don't we just run watch?

Reply:
I tried using watch, but watch never completes since it's on going and that prevents the pre-launch task from "completing".